client/my-sites/domains/domain-management/list/list-header.jsx

```diff
@@ -54,7 +82,12 @@ class ListHeader extends React.PureComponent {
 		return (
 			<>
 				<div className="list__domain-link" />
-				<div className="list__domain-transfer-lock">
+				<div
+					className="list__domain-transfer-lock"
+					onMouseEnter={ () => this.enableTooltip( 'transferLock' ) }
+					onMouseLeave={ () => this.disableTooltip( 'transferLock' ) }
```

Comment:
I think it's prob better to create a class method to do this and pass that in here since this will create a new anonymous function on each render of the component which will cause re-renders for each of these header div's even though their props haven't actually changed.

Reply:
The reason why I chose arrow functions instead of bound ones in here was because the argument was only available in the context (e.g `'transferLock'` string). 

So, to avoid that, I added some (_hopefully?_) little complexity around the header items and their data (label, description and etc), causing fewer renders and tidying up things.